### ISSUE TITLE
Extraction without group folder

### ISSUE THREAD

**Issue by gcxd68** [twogood/unshield#126]
Hi,

Thank you for all the hard work. Il was looking for a way to extract files without creating group folders.

Example : cabinet files contains 2 groups : DATA and EXECUTABLE. If I extract both towards an INSTALL folder, DATA and EXECUTABLE folders are created. I would like to get the files directly in the INSTALL folder and skip the group folder creation.

Is that even possible ?

Thank you,

Greg

**Comment by twogood** [comment]
Thank you Greg for the kind words! 

I don't have an obvious solution but maybe if you call it twice: once with `-g DATA` and once with `-g EXECUTABLE`?

**Comment by gcxd68** [comment]
Thank you for this quick response !

I already tried it, the DATA and EXECUTABLE folders still get created in the INSTALL one. Even with `-g DATA` or `-g EXECUTABLE`

**Comment by twogood** [comment]
If there is no directory structure, maybe `-j` will work? Otherwise I'm afraid you will have to relocate the files after extracting.

**Comment by gcxd68** [comment]
Ok, I'll have a look, thank you !

**Comment by twogood** [comment]
The reason for creating these directories, IIRC, is that sometimes files with the same name can exist in multiple groups or components.

**Comment by gcxd68** [comment]
Yes I understand. I'll try to explain my goal : I'm making an installer for an old software that won't install anymore on recent computers (but still works). Because of these files extracting to this "group" folder, I have to extract them to a temporary folder and then copy them again to the destination. Not optimal, but it works. Thanks again.

**Comment by twogood** [comment]
Yeah @gcxd68 that is a very typical use case, but I hope that moving the files to the right places (don't copy, that is much slower) with a script afterwards is not too much trouble.
